src/org/opensolaris/opengrok/configuration/messages/Message.java

```diff
@@ -57,95 +57,48 @@
  */
 public abstract class Message implements Comparable<Message> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(Message.class);
+    public static final String DELIMITER = "\0";
```

Comment:
is it safe that the delimiter can not occur in the middle of the json?

Reply:
<https://www.json.org> I don't see `\0` there...

Reply:
perfect